_includes/v22.2/ui/active-transaction-executions.md

```diff
@@ -0,0 +1,46 @@
+## Active Executions table
+
+Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.
+
+The Active Executions table gives details for each transaction fingerprint in the transaction:
+
+Column | Description
+-----|------------
+Transaction Execution ID | The execution ID of the transaction.
+Most Recent Statement | The most recently executed statement in the transaction.
+Status | The status of the execution: Preparing, Waiting, or Executing.
+Start Time (UTC) | The time the execution started.
```

Comment:
Same comment as above re: timestamp.

Reply:
Fixed.

Reply:
This looks unchanged on my end (still reads "The time the...")